ProtoWorkspace/Prelude.fs

```diff
@@ -110,229 +124,224 @@ module Array =
+    
+    let (|StartsWith|_|) pattern value = 
         if String.IsNullOrWhiteSpace value then None
-        elif value.StartsWith pattern then Some ()
+        elif value.StartsWith pattern then Some()
         else None
-
-    let (|Contains|_|) pattern value =
+    
+    let (|Contains|_|) pattern value = 
         if String.IsNullOrWhiteSpace value then None
-        elif value.Contains pattern then Some ()
+        elif value.Contains pattern then Some()
         else None
     
     open System.IO
-
-    let getLines (str: string) =
+    
+    let getLines (str : string) = 
         use reader = new StringReader(str)
-        [|  let line = ref (reader.ReadLine())
-            while isNotNull (!line) do
-                yield !line
-                line := reader.ReadLine()
-            if str.EndsWith "\n" then
-            // last trailing space not returned
-            // http://stackoverflow.com/questions/19365404/stringreader-omits-trailing-linebreak
-                yield String.Empty
-        |]
-
-    let getNonEmptyLines (str: string) =
+        [| let line = ref (reader.ReadLine())
+           while isNotNull (!line) do
+               yield !line
+               line := reader.ReadLine()
+           if str.EndsWith "\n" then 
+               // last trailing space not returned
+               // http://stackoverflow.com/questions/19365404/stringreader-omits-trailing-linebreak
+               yield String.Empty |]
+    
+    let getNonEmptyLines (str : string) = 
         use reader = new StringReader(str)
-        [|  let line = ref (reader.ReadLine())
-            while isNotNull (!line) do
-                if (!line).Length > 0 then yield !line
-                line := reader.ReadLine()
-        |]
-
+        [| let line = ref (reader.ReadLine())
+           while isNotNull (!line) do
+               if (!line).Length > 0 then yield !line
+               line := reader.ReadLine() |]
+    
     /// Parse a string to find the first nonempty line
     /// Return null if the string was null or only contained empty lines
-    let firstNonEmptyLine (str: string) =
-        use reader = new StringReader (str)
-        let rec loop (line:string) =
-            if isNull line then None 
-            elif  line.Length > 0 then Some line
+    let firstNonEmptyLine (str : string) = 
+        use reader = new StringReader(str)
+        
+        let rec loop (line : string) = 
+            if isNull line then None
+            elif line.Length > 0 then Some line
             else loop (reader.ReadLine())
         loop (reader.ReadLine())
 
 open System.Text
+
 [<CompilationRepresentation(CompilationRepresentationFlags.ModuleSuffix)>]
-module StringBuilder =
+module StringBuilder = 
     /// Pipelining function for appending a string to a stringbuilder
-    let inline append (str:string) (sb:StringBuilder) = sb.Append str
-
+    let inline append (str : string) (sb : StringBuilder) = sb.Append str
```

Comment:
adding spaces in every instance like `(str:string)` to `(str : string)` is a waste. it doesn't make it any easier to read